test/Compiler.Dynamic.Tests/assets/test03/Site.Master

```diff
@@ -0,0 +1,21 @@
+<%@ Master Language="C#" AutoEventWireup="true" CodeBehind="Site.master.cs" Inherits="SystemWebUISample.SiteMaster" %>
```

Comment:
The test is failing for case sensitive "master" (CodeBehind="Site.Master.cs").. (not sure, if this enforced in framework, if not create follow-up issue), else fix in the test data

Reply:
that's because it runs on linux where the filesystem has case sensitivity